### ISSUE TITLE
Syntac highlighting adds an odd box around lower-case unicode letters

### ISSUE THREAD

**Issue by RalfJung** [coq-community/vscoq#905]
I often use unicode letters in my Coq identifers, to better match what we do on paper. This works fine in Coq, but the syntax highlighting of this extension draws a box around some these characters:

![image](https://github.com/user-attachments/assets/8adaed43-39c7-4089-b130-98b0cafc65bf)

This makes them look like some sort of error, and also makes them hard to read.

In a plain text file these letters look just right, so it seems to be something specific to what VsCoq does in `.v` files.

**Comment by 4ever2** [comment]
I don't think this is anything specific to VsCoq, it seems to be because of the `editor.unicodeHighlight.ambiguousCharacters` setting. When enabled it will highlight any ambiguous unicode characters in all non plaintext/markdown files.

**Comment by RalfJung** [comment]
Thanks for pointing that out! That helps indeed. :)  I have no clue why vscode thinks σ could be confused with another charcacter (but, say, Λ could not)... but now I can at least disabled this behavior.
